build_n_deploy/webpack/webpack.common.js

```diff
@@ -35,6 +35,10 @@ module.exports = {
                     transpileOnly: true,
                 },
             },
+            {
+                test: /\.svg$/,
+                loader: 'svg-inline-loader',
+            },
```

Comment:
@charliemidtlyng Måtte legge til denne for å få `nav-frontend-tabell-style` til å rulle. Ser det greit ut?